app/redux/actions/async.js

```diff
@@ -733,7 +733,7 @@ export function requestPasswordReset(api, email) {
   return (dispatch) => {
     dispatch(sync.requestPasswordResetRequest());
 
-    api.user.requestPasswordReset(email, function(err) {
+    api.user.requestPasswordReset(email, function (err) {
```

Comment:
line #258 the same kind of function was transformed to an arrow function, should we do the same here ?
```suggestion
    api.user.requestPasswordReset(email, (err) => {
```